lib/dradis/plugins/netsparker/gem_version.rb

```diff
@@ -9,7 +9,7 @@ def self.gem_version
       module VERSION
         MAJOR = 3
         MINOR = 15
-        TINY = 0
+        TINY = 1
```

Comment:
No version bump needed. We won't produce a full release until 3.16.0.

Reply:
@brianp Are you saying this should go back to 0 or stay at 1? Can you add a suggestion?

Reply:
Correct, staying at 0 is fine.

```suggestion
        TINY = 0
```